### ISSUE TITLE
provides dropped out of META.yaml in 2.59

### ISSUE THREAD

**Issue by toddr** [lstein/Perl-GD#20]
``` diff

diff --git a/META.yml b/META.yml
index f8f02021a..c9e045951 100644
--- a/META.yml
+++ b/META.yml
@@ -1,38 +1,26 @@
 ---
-abstract: 'Perl interface to the gd2 graphics library'
+abstract: 'Interface to Gd Graphics Library'
 author:
-  - 'Lincoln Stein <lincoln.stein@gmail.com>'
+  - 'Lincoln Stein (lstein@cshl.org)'
 build_requires:
-  ExtUtils::CBuilder: '0'
+  ExtUtils::MakeMaker: '0'
 configure_requires:
-  Module::Build: '0.42'
+  ExtUtils::MakeMaker: '0'
 dynamic_config: 1
-generated_by: 'Module::Build version 0.421, CPAN::Meta::Converter version 2.142690'
-license: perl
+generated_by: 'ExtUtils::MakeMaker version 7.24, CPAN::Meta::Converter version 2.150010'
+license: unknown
 meta-spec:
   url: http://module-build.sourceforge.net/META-spec-v1.4.html
   version: '1.4'
 name: GD
-provides:
-  GD:
-    file: lib/GD.pm
-    version: '2.56'
-  GD::Group:
-    file: lib/GD/Group.pm
-    version: '1'
-  GD::Polygon:
-    file: lib/GD/Polygon.pm
-  GD::Polyline:
-    file: lib/GD/Polyline.pm
-    version: '0.2'
-  GD::Simple:
-    file: lib/GD/Simple.pm

**Comment by toddr** [comment]
Also is () valid for email addresses?

**Comment by toddr** [comment]
Meta.json is also missing git URL data.

``` diff
diff --git a/META.json b/META.json
index 0051a81b2..e3a17a30b 100644
--- a/META.json
+++ b/META.json
@@ -1,65 +1,48 @@
 {
-   "abstract" : "Perl interface to the gd2 graphics library",
+   "abstract" : "Interface to Gd Graphics Library",
    "author" : [
-      "Lincoln Stein <lincoln.stein@gmail.com>"
+      "Lincoln Stein (lstein@cshl.org)"
    ],
    "dynamic_config" : 1,
-   "generated_by" : "Module::Build version 0.421",
+   "generated_by" : "ExtUtils::MakeMaker version 7.24, CPAN::Meta::Converter version 2.150010",
    "license" : [
-      "perl_5"
+      "unknown"
    ],
    "meta-spec" : {
       "url" : "http://search.cpan.org/perldoc?CPAN::Meta::Spec",
       "version" : "2"
    },
    "name" : "GD",
+   "no_index" : {
+      "directory" : [
+         "t",
+         "inc"
+      ]
+   },
    "prereqs" : {
       "build" : {
          "requires" : {
-            "ExtUtils::CBuilder" : "0"
+            "ExtUtils::MakeMaker" : "0"
          }
       },
       "configure" : {
          "requires" : {
-            "Module::Build" : "0.42"
+            "ExtUtils::MakeMaker" : "0"
          }
       },
       "runtime" : {
          "requires" : {
+            "ExtUtils::PkgConfig" : "0",
             "Math::Trig" : "0"
          }
       }
    },
-   "provides" : {
-      "GD" : {
-         "file" : "lib/GD.pm",
-         "version" : "2.56"
-      },
-      "GD::Group" : {
-         "file" : "lib/GD/Group.pm",
-         "version" : "1"
-      },
-      "GD::Polygon" : {
-         "file" : "lib/GD/Polygon.pm"
-      },
-      "GD::Polyline" : {
-         "file" : "lib/GD/Polyline.pm",
-         "version" : "0.2"
-      },
-      "GD::Simple" : {
-         "file" : "lib/GD/Simple.pm"
-      }
-   },
    "release_status" : "stable",
    "resources" : {
       "license" : [
          "http://dev.perl.org/licenses/"
       ]
    },
-   "version" : "2.56",
-   "x_repository" : {
-      "type" : "git",
-      "url" : "git://github.com/lstein/Perl-GD.git",
-      "web" : "https://github.com/lstein/Perl-GD"
-   }
+   "version" : "2.59",
+   "x_serialization_backend" : "JSON::PP version 2.27300_01"
 }

**Comment by rurban** [comment]
This seems to be a EUMM bug. Never saw provides: with EUMM

**Comment by chorny** [comment]
AFAIK, EUMM does not generate "provides"

**Comment by toddr** [comment]
Fair enough. You can do the github URL though.

https://github.com/toddr/perl-net-jabber-bot/blob/master/Makefile.PL#L38

**Comment by rurban** [comment]
Fixed with 2.60
